code/__DEFINES/jobs.dm

```diff
@@ -6,6 +6,8 @@
 #define JOB_UNAVAILABLE_SLOTFULL 5
 /// Job unavailable due to incompatibility with an antag role.
 #define JOB_UNAVAILABLE_ANTAG_INCOMPAT 6
+/// Job unavailable due to your character being loli
```

Comment:
this is not acceptable verbiage for this codebase

Reply:
Bro

Reply:
Fine, i'll fix it

Reply:
WHAT